client/src/components/screens/Patient/components/PatientHeader/components/ProbandHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import intl from 'react-intl-universal';
+import { Col, Tag, Typography } from 'antd';
+import { ParsedPatientData } from 'helpers/providers/types';
+
+import FemaleIcon from 'components/Assets/Icons/FemaleIcon'
+import MaleIcon from 'components/Assets/Icons/MaleIcon';
+import UnknowGenderIcon from 'components/Assets/Icons/UnknowGenderIcon'
+
+import './styles.scss';
+  
+enum Gender {
+MALE = 'male',
+FEMALE = 'female',
+UNKNOWN = 'unknown',
+}  
+
+const getGenderIcon = (gender: string) => {
+  switch (gender) {
```

Comment:
gender: string should be gender: Gender unless your are not sure of the inputted type.

Reply:
☝️